R/transform.R

```diff
@@ -47,22 +47,27 @@ usmap_transform <- function(data,
                             input_names = c("lon", "lat"),
                             output_names = c("x", "y")) {
   # check for maptools
-  if (!requireNamespace("maptools", quietly = TRUE)) {
-    stop("`maptools` must be installed to use `usmap_transform`.
-         Use: install.packages(\"maptools\") and try again.")
-  }
+#  if (!requireNamespace("maptools", quietly = TRUE)) {
+#    stop("`maptools` must be installed to use `usmap_transform`.
+#         Use: install.packages(\"maptools\") and try again.")
+#  }
```

Comment:
Since `maptools` is being removed I think we can remove the check entirely instead of just commenting it out

Reply:
Yes, that makes sense.